test/replaceJQueryTest.ts

```diff
@@ -0,0 +1,92 @@
+import {
```

Comment:
hm I would see this as part of addMinissingDependencies test

and the test files could go into a subfolder

Reply:
done